zerver/lib/actions.py

```diff
@@ -189,6 +190,12 @@ def private_stream_user_ids(stream_id: int) -> Set[int]:
     subscriptions = get_active_subscriptions_for_stream_id(stream_id)
     return {sub['user_profile_id'] for sub in subscriptions.values('user_profile_id')}
 
+def public_stream_user_ids(stream: Stream) -> Set[int]:
+    guest_subscriptions = get_active_subscriptions_for_stream_id(
+        stream.id).filter(user_profile__is_guest=True)
+    guest_subscriptions = {sub['user_profile_id'] for sub in guest_subscriptions.values('user_profile_id')}
+    return set(active_user_ids(stream.realm_id)) | guest_subscriptions
```

Comment:
This first part of the `return` statement should be `active_non_guest_user_ids`, not `active_user_ids`, right?